### DISCUSSION TITLE
图片上传失败: 82 是什么错误？

### DISCUSSION THREAD

**Discussion by Krimiston** [Mrs4s/go-cqhttp#1736] [Q&A]
我的 Bot 在往群聊中发图片的时候 gocq 报了这个错误：
```
图片上传失败: upload failed: upload failed: 82
```
API 返回了 `SEND_MSG_API_ERROR`。
请问关于82这个错误码有什么说法吗？

**Comment by wdvxdr1123** [comment]
可能图片太涩了

**Reply by Krimiston** [reply]
如果是因为图片太色，前端会直接说“账号可能被风控”。
而且，图片不色，图片内容是摸鱼人日历。
